.github/workflows/publish-new-map.yml

```diff
@@ -15,6 +15,10 @@ jobs:
       run: |
         yarn install --frozen-lockfile
         node ./map-utilities/generate-map.js
+        node ./map-utilities/generate-npcs.mjs
+      env:
+        MONGO_API_KEY: ${{ secrets.MONGO_API_KEY }}
```

Comment:
Does someone else have access to this secret in case something needs fixing?

Reply:
Khaseem (@Log-Wall) can provide it when needed :thinking: though it's a read-only token with public data, so maybe we can just add it here?